android/src/main/kotlin/com/albertalrisa/flutter/plugins/shareapi/intents/SystemUI.kt

```diff
@@ -23,7 +23,7 @@ class SystemUI(authority_name: String, registrar: Registrar, activity: Activity)
                 shareFile(arguments["file_url"], arguments["type"]!!, arguments["prompt"], result)
             }
             "shareImage" -> {
-                shareImage(arguments["image_url"], arguments["type"]!!, arguments["prompt"], result)
+                shareImage(arguments["image_url"], arguments["type"]!!, arguments["prompt"], arguments["text"]!!, result)
```

Comment:
Will this be safe?
Because you are using the not-null assertion operator (`!!`) but its value can be null.

I think it is better to remove the `!!` operator because your function signature allow `String?`.

Reply:
For sure, didn't quite understand that !! assertion, would the change be removing "!!" only?
Thanks.

Reply:
Yeah, I think that will be it.

The problem is that the text should be optional if you're sharing images, right?
If you put that there, then if the text passed is `null`, Kotlin will produce a `NullPointerException` and crashes the plugin.

Reply:
Done